src/1_create_canonical.jl

```diff
@@ -116,8 +116,11 @@ output_features[!, contains.(string.(typeof.(eachcol(output_features))), "String
 # Correct incorrect GBRMPA_ID to match cots_priority data
 output_features.GBRMPA_ID = ifelse.((output_features.GBRMPA_ID .== "20198"), "20-198", output_features.GBRMPA_ID)
 
+# Reproject output_features from GDA94 EPSG4283 to GDA2020 EPSG7844 to match GBRMPA geohub data
+output_features.geometry = AG.reproject(output_features.geometry, GI.crs(output_features[1,:geometry]), EPSG(7844); order=:trad)
+
 # Save geopackage
-GDF.write(joinpath(OUTPUT_DIR, "rrap_canonical_$(Dates.format(now(), DATE_FORMAT)).gpkg"), output_features; crs=GFT.EPSG(4326))
+GDF.write(joinpath(OUTPUT_DIR, "rrap_canonical_$(Dates.format(now(), DATE_FORMAT)).gpkg"), output_features; crs=GFT.EPSG(7844))
```

Comment:
`EPSG(7844)` is repeated several times, and it will be a pain if we later decide to standardize to another datum.

I would set this as a global variable in `common.jl`. 
This way it only needs to be changed in one place if we need to adjust later.